.env.example

```diff
@@ -57,6 +57,8 @@ GITHUB_TOKEN={fill me in}
 
 # FLOWDOCK_API_TOKEN= # required for the flowdock integration user mention autocomplete
 
+# SLACK_API_TOKEN= # only required for the slack integration to fetch users in the buddy approval request form (when BUDDY_CHECK_FEATURE=1)
```

Comment:
I think we sh flowdock and this one optional ... also flowdock needs this `only when buddy` comment